cli/src/Morphir/Web/DevelopApp.elm

```diff
@@ -604,13 +642,15 @@ viewBody model =
         IRLoaded (Library packageName _ packageDef) ->
             viewHome model packageName packageDef
 
-
+{-| Display the home UI
+-}
 viewHome : Model -> PackageName -> Package.Definition () (Type ()) -> Element Msg
 viewHome model packageName packageDef =
     let
         gray =
             rgb 0.9 0.9 0.9
 
+        {-| Styles to make the module tree and the definition list scrollable -}
```

Comment:
Looks like the Elm parser doesn't like these multi-line comments inside a value definition. Probably they are only allowed at the top level. You can replace it with single-line comments (`--` in the front)